---
sensor_msgs/Image[] color_frames
sensor_msgs/Image[] depth_frames
# sensor_msgs/PointCloud2[] point_cloud